applications/app/views/atomEmbed.scala.html

```diff
@@ -15,4 +15,15 @@
     @page.body
     @fragments.atomPageFoot(page)
 </body>
+    <script>
+        // TODO: call this when the atom is ready
+        window.parent.postMessage(
+            {
+                sentinel: 'amp',
+                type: 'embed-size',
+                height: document.body.scrollHeight,
+            },
+            '*'
+        );
```

Comment:
We should be sending this message when the interactive atom is "ready" in order for the size to be accurate (i.e. after loading data in via ajax).